getting-started/templates/systemlink-values.yaml

```diff
@@ -897,3 +897,51 @@ comments:
   ## Maximum number of comments supported per delete comments request.
   ##
   maximumCommentsPerDeleteRequest: 1000
+
+## Configuration for work order service.
+##
+workorder:
+  ## Set to true to enable the deployment of the work order and lab management ui service. By default, 
+  ## this is disabled and the work order and lab management ui service are not deployed.
+  ##
+  enabled: false
```

Comment:
Consider adding an ATTENTION tag to this value if this is a decision that should be made by the customer at installation time.

Reply:
Added the ATTENTION tag.